src/components/post/Post.js

```diff
@@ -171,15 +174,17 @@ class Post extends Component {
             <Grid>
               <div
                 id="notebook-container"
-                dangerouslySetInnerHTML={{ __html: upload.html.final_html }}
+                dangerouslySetInnerHTML={{ __html: upload.html.final_html || upload.html}}
```

Comment:
what's the or statement for?

Reply:
the endpoint to retrieve post when you click showmore doesn't have **final_html prop**